manual/cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/MergeTest.scala

```diff
@@ -231,7 +231,7 @@ return r""",
 merge (oliver)-[:DIRECTED]->(movie:Movie)<-[:ACTED_IN]-(reiner)
 return movie""",
       optionalResultExplanation = "In our example graph, 'Oliver Stone' and 'Rob Reiner' have never worked together. When we try to +MERGE+ a " +
-        "movie between them, Cypher will not use any of the existing movies already connected to either person. Instead, " +
+        "movie between them, Neo4j will not use any of the existing movies already connected to either person. Instead, " +
```

Comment:
I've been treating Cypher as a language, and as such not an active participant in query running.